vdb-bench-assembly/dev-server/json/vdbs.json

```diff
@@ -1,15 +1,15 @@
 {
     "vdbs": [
         {
-            "id": "portfolio",
+            "id": "Portfolio",
             "_links": [
                 {
                     "rel": "self",
-                    "href": "/vdbs/portfolio"
+                    "href": "/vdbs/Portfolio"
                 },
```

Comment:
Not sure what purpose it would serve. The vdbs resource is a 'flat' structure (well 1-level down). A parent link would point to /vdbs which a client would already have seeing at they used it to GET the collection of vdbs in the first place. I could put it in but just seems a little redundant.